code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -419,6 +419,15 @@
 	heat_protection = CHEST|GROIN|ARMS|LEGS
 	cold_protection = CHEST|GROIN|ARMS|LEGS
 
+/obj/item/clothing/suit/jacket/leather/duster/armoured
```

Comment:
Is this strictly necessary? We have a black armored duster already, which is a subtype of the armor vest (and therefore also benefits from the suit storage permissions that come with it). Having a brown version (admittedly with weaker armor) seems redundant. Additionally, if you do keep it, change all new instances of "armoured" to "armored" for consistency. We may as well standardize at some point and most other clothing items use the american spelling.